bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -223,15 +223,29 @@ spec:
     Remediation” API to instantiate the configured remediation template which
     specifies the mechanism/controller to be used.
 
-    ### Failure handling: Self Node Remediation
-    By default NHC depends on the “Self Node Remediation” (SNR) operator, which
-    is installed automatically.
+    ### Failure handling: External Remediators
+    The Medik8s team developed few remediators for handling node failure:
+    - Self Node Remediation (SNR)
+    - Fence Agents Remediation (FAR)
+    - Machine Deletion Remediation (MDR)
+
+    By default NHC depends and use SNR operator, which is installed automatically, as its default remediator.
+
+    #### Self Node Remediation (SNR)
     SNR uses watchdog timers and heuristics to ensure nodes enter a safe state
     (no longer hosting workloads) within a known and finite period of time,
-    before signaling to the system that all Pods and VolumeAttachments on the
-    failed Node are no longer active and can be relocated elsewhere.
+    before signaling to the system that all Pods on the failed Node are no longer active
+    and can be relocated elsewhere.
     In the case of transient errors, the watchdog’s actions will also result in
     the node rebooting and rejoining the cluster - restoring capacity.
+
+    #### Fence Agents Remediation (FAR)
+    FAR uses well-known agents to fence and remediate unhealthy nodes.
```

Comment:
I think this line can be a bit misleading.
The way I read it it can be implied that the fencing and remediation is done by the fence agents, where in practice the fence agents are only used for fencing.
Triggering the workload relocation (which is a major part of  the remediation process) isn't done by the fence agents.